apps/omg_watcher/lib/exit_processor.ex

```diff
@@ -361,12 +359,13 @@ defmodule OMG.Watcher.ExitProcessor do
     {:reply, response, state}
   end
 
-  def handle_call({:create_challenge, Utxo.position(blknum, txindex, oindex) = exiting_utxo_pos}, _from, state) do
+  def handle_call({:create_challenge, Utxo.position(blknum, _txindex, oindex) = exiting_utxo_pos}, _from, state) do
     with spending_blknum_response <- exiting_utxo_pos |> Utxo.Position.to_db_key() |> OMG.DB.spent_blknum(),
-         %{txhash: txhash} <- OMG.Watcher.DB.Transaction.get_by_position(blknum, txindex),
-         {:ok, exit_id} <- OMG.Eth.RootChain.get_standard_exit_id(txhash, oindex),
-         {:ok, raw_spending_proof, exit_info} <-
-           Core.get_challenge_data(spending_blknum_response, exiting_utxo_pos, state) do
+         {:ok, {block_hash, _block_timestamp}} <- Eth.RootChain.get_child_chain(blknum),
```

Comment:
you can fetch the block hash consistently with the other occurrence here: `{:ok, hashes} = OMG.DB.block_hashes([raw_blknum])` - see below